src/pudl/package_data/settings/etl_fast.yml

```diff
@@ -55,7 +55,7 @@ datasets:
         - generators_eia860
         - ownership_eia860
       years: [2020]
-      eia860m: true
+      eia860m: false
```

Comment:
with the integration of the left out commit, these have now been changed.

I did include both 2020 and 2021 because it felt a lil weird to only have the CI run on early release data... not sure how you feel about that or if it is viable in the short run.